python/cudf/cudf/core/index.py

```diff
@@ -343,6 +343,10 @@ def astype(self, dtype, copy: bool = True):
     def drop_duplicates(self, keep="first"):
         return self
 
+    @_cudf_nvtx_annotate
+    def duplicated(self, keep="first"):
```

Comment:
Does this inherit a docstring? Or does it need something?

Reply:
This will inherit docstring `BaseIndex.duplicated`